frontend/javascripts/test/puppeteer/dataset_rendering.screenshot.ts

```diff
@@ -1,3 +1,4 @@
+import "test/mocks/lz4";
```

Comment:
Good question :see_no_evil: I assume that the import dependencies changed somehow so that the lz4 dependency is imported now (which fails on node). I didn't bother to investigate further. Hope this is alright!